apis.json

```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "category": "Animals",
+        "apis": [
+            {
+                "name": "IUCN",
+                "description": "IUCN Red List of Threatened Species",
+                "auth": "token",
+                "link": "http://apiv3.iucnredlist.org/api/v3/docs"
+            }, {
+                "name": "Petfinder",
+                "description": "Adoption",
+                "auth": "no",
+                "link": "https://www.petfinder.com/developers/api-docs/"
+            }, {
+                "name": "RescueGroups",
+                "description": "Adoption",
+                "auth": "no",
+                "link": "https://userguide.rescuegroups.org/display/APIDG/API+Developers+Guide+Home"
+            }
+        ]
+    }, {
+        "category": "Anime",
```

Comment:
Wanted to point out that the JSON format used here is different than what was last recommended in #259: 
```
[
    {
        "name": "IUCN",
        "description": "IUCN Red List of Threatened Species",
        "auth": "token",
        "link": "http://apiv3.iucnredlist.org/api/v3/docs",
        "categories": ["Animals"]
    },
```
The difference being how the category is used.  The first format opens us to duplicate entries if an API is put in multiple categories creating a risk that they aren't identical.  i.e. the link gets updated in one entry but not the duplicate.  

Are we ok with that?  I just recently started following so I'm not sure what the protocol is for making decisions on this project.